app/components/isp-jam-auth/template.hbs

```diff
@@ -13,7 +13,7 @@
         </div>
       </div>
       <div class="form-group">
-        <button disabled={{if (or authenticating (not (and password username))) true false}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
+        <button disabled={{disabled}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
```

Comment:
Can you please clarify where this property is defined? It does not appear to be part of the matching component.js, nor is it a variable passed in.